phpBB/includes/acp/acp_extensions.php

```diff
@@ -276,6 +276,12 @@ public function main_mode($id, $mode)
 				{
 					$this->template->assign_var('MIGRATOR_ERROR', $e->getLocalisedMessage($this->user));
 				}
+				catch (\Exception $e)
+				{
+					$stack_trace = phpbb_filter_root_path(str_replace("\n", '<br>', $e->getTraceAsString()));
```

Comment:
I'd rather move the HTML handling to the template files. So do the filtering but then assign a `MIGRATOR_STACK_TRACE` and do this replacement with twig (https://twig.symfony.com/doc/3.x/filters/nl2br.html)